doc/Rules.md

```diff
@@ -1373,7 +1373,7 @@ or match the given style if the rule is so configured:
 ```
 
 Note: by default, this rule is configured to just require that all horizontal
-rules in the document are the same, and will trigger if any of the horizontal
+rules in the document are the same and will trigger if any of the horizontal
```

Comment:
I think the comma should be kept, as it helps pace the reading of this long sentence.

Reply:
I like this one better the new way.

Reply:
Compound sentences introducing a new subject need a comma. In this scenario, a new subject is not introduced.